package.json

```diff
@@ -57,31 +57,30 @@
   },
   "dependencies": {
     "@oclif/command": "^1.0",
-    "@oclif/config": "^1.12.11",
+    "@oclif/config": "^1.12",
     "@oclif/plugin-help": "^2.0",
-    "@stoplight/json": "^2.1.0",
-    "@stoplight/json-ref-resolver": "^1.5.0",
-    "@stoplight/types": "^7.0.1",
-    "@stoplight/yaml": "^2.3.0",
-    "ajv": "6.x.x",
+    "@stoplight/json": "^2.2.2",
+    "@stoplight/json-ref-resolver": "^1.5",
+    "@stoplight/types": "^7.0",
+    "@stoplight/yaml": "^2.5",
+    "ajv": "^6",
```

Comment:
Is this intentional here? It would allow 7, 8 or 9 which could have breaking changes. We can avoid that with `^6.0`.

Reply:
It actually only allows version 6 range (`^` = minor/patch can change). I recently found https://semver.npmjs.com/ which is awesome to understand this stuff.

Reply:
Stupid memory. I was mixing it up with ~ sorry.